app/src/main/kotlin/com/adesso/movee/widget/expandabletextview/ExpandableTextView.kt

```diff
@@ -0,0 +1,115 @@
+package com.adesso.movee.widget.expandabletextview
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import androidx.core.widget.TextViewCompat
+import androidx.databinding.BindingAdapter
+import com.adesso.movee.R
+import com.adesso.movee.databinding.ViewExpandableTextBinding
+import com.adesso.movee.internal.extension.gone
+import com.adesso.movee.internal.extension.show
+
+@BindingAdapter("etv_content")
+fun setContent(view: ExpandableTextView, content: String?) {
+    view.content = content
+}
+
+class ExpandableTextView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null
+) : LinearLayout(context, attrs) {
+
+    private var collapseLines = -1
+    private var collapsedToggleText: String? = null
+    private var expandedToggleText: String? = null
+    private var expanded = false
+    private val binder = ViewExpandableTextBinding.inflate(
+        LayoutInflater.from(context),
+        this,
+        true
+    )
+    var content: String? = null
+        set(value) {
+            field = value
+            with(binder.textViewContent) {
+                text = content
+                post {
+                    if (collapseLines > 0 && lineCount > collapseLines) {
+                        binder.textViewToggle.show()
+                        collapse()
+                    } else {
+                        binder.textViewToggle.gone()
+                    }
+                }
+            }
+        }
+
+    init {
+
+        orientation = VERTICAL
+
+        readAttributes(attrs)
+
+        binder.textViewToggle.setOnClickListener {
+            if (expanded) {
+                collapse()
+            } else {
+                expand()
+            }
```

Comment:
statements can be one-line without curly bracket if both of the branches consist of one-line expressions